css/properties/dynamic-range-limit.json

```
@@ -62,6 +62,38 @@
             }
           }
         },
+        "dynamic-range-limit-mix": {
+          "__compat": {
+            "description": "`dynamic-range-limit-mix()` function",
```

In the context of the CSS property, the support should probably refer to the value type, not the function itself:

```suggestion
        "dynamic-range-limit-mix_values": {
          "__compat": {
            "description": "`<dynamic-range-limit-mix()>` values",
```

(Note: I'm suggesting the `_value` prefix as a compromise between @Elchi3's [comment](https://github.com/mdn/browser-compat-data/pull/28027#pullrequestreview-3283901044) and [mine](https://github.com/mdn/browser-compat-data/pull/28027#issuecomment-3352365184).)